pkg/logs/input/docker/json_parser_test.go

```diff
@@ -0,0 +1,42 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2019 Datadog, Inc.
+
+package docker
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/DataDog/datadog-agent/pkg/logs/message"
+)
+
+func TestJSONParser(t *testing.T) {
```

Comment:
could you test the case when 
* a field is empty (if `log` field is empty, is that a valid case?)
* a field absent

Reply:
no need to test when a field is absent because by default, when a field is absent, go would put a null `value`, same for `unrecognised` field, by design it won't fail.
If the log is empty, the parsing will succeed as well.